logstash_async/worker.py

```diff
@@ -202,6 +208,7 @@ def _flush_queued_events(self, force=False):
             else:
                 self._delete_queued_events_from_database()
                 self._reset_flush_counters()
+                self._flush_event.clear()
```

Comment:
I'm wondering if it might be better to set the event clear call a bit ealier, maybe already at line 200 or even 187:
if flushing the events doesn't work because of a database lock or network errors or whatever other reason, we maybe should clear the flush request anyway and try again on the next explicit flush request or if the next interval is reached.
I could imagine otherwise we would try to flush very often in case of temporary network errors or such and so cause a lot of unnecessary work for the thread. On the other hand, it depends on the use case. If it is essential for the application to ensure events are flushed quickly and without unnecessary delays, trying often might be preferred.

What do you think?

Reply:
Hm, yeah, it seems reasonable!

I'll do it, but maybe we should point on that behaviour in documentation?

Reply:
Agreed. Do you want to do it? If not, no problem, I can do it also post-merge.

Reply:
Hm, I pretty bad with english yet, so I'll be glad if u do it for me :)

Thank you!